xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/dto/trade/BittrexOrder.java

```diff
@@ -0,0 +1,224 @@
+      @JsonProperty("Closed") Date closed,
+      @JsonProperty("IsOpen") Boolean isOpen,
+      @JsonProperty("Sentinel") String sentinel,
+      @JsonProperty("CancelInitiated") Boolean cancelInitiated,
+      @JsonProperty("ImmediateOrCancel") Boolean immediateOrCancel,
+      @JsonProperty("IsConditional") Boolean isConditional,
+      @JsonProperty("Condition") String condition,
+      @JsonProperty("ConditionTarget") String conditionTarget) {
+    this.accountId = accountId;
+    this.orderUuid = orderUuid;
+    this.exchange = exchange;
+    this.type = type;
+    this.quantity = quantity;
+    this.quantityRemaining = quantityRemaining;
+    this.limit = limit;
+    this.reserved = reserved;
+    this.reserveRemaining = reserveRemaining;
+    this.commissionReserved = commissionReserved;
+    this.commissionReserveRemaining = commissionReserveRemaining;
+    this.commissionPaid = commissionPaid;
+    this.price = price;
+    this.pricePerUnit = pricePerUnit;
+    this.opened = opened;
+    this.closed = closed;
+    this.isOpen = isOpen;
+    this.sentinel = sentinel;
+    this.cancelInitiated = cancelInitiated;
+    this.immediateOrCancel = immediateOrCancel;
+    this.isConditional = isConditional;
+    this.condition = condition;
+    this.conditionTarget = conditionTarget;
+  }
+
+  public String getAccountId() {
+    return accountId;
+  }
+
+  public String getOrderUuid() {
+    return orderUuid;
+  }
+
+  public String getExchange() {
+    return exchange;
+  }
+
+  public String getType() {
+    return type;
+  }
+
+  public BigDecimal getQuantity() {
+    return quantity;
+  }
+
+  public BigDecimal getQuantityRemaining() {
+    return quantityRemaining;
+  }
+
+  public BigDecimal getLimit() {
+    return limit;
+  }
+
+  public BigDecimal getReserved() {
+    return reserved;
+  }
+
+  public BigDecimal getReserveRemaining() {
+    return reserveRemaining;
+  }
+
+  public BigDecimal getCommissionReserved() {
+    return commissionReserved;
+  }
+
+  public BigDecimal getCommissionReserveRemaining() {
+    return commissionReserveRemaining;
+  }
+
+  public BigDecimal getCommissionPaid() {
+    return commissionPaid;
+  }
+
+  public BigDecimal getPrice() {
+    return price;
+  }
+
+  public BigDecimal getPricePerUnit() {
+    return pricePerUnit;
+  }
+
+  public Date getOpened() {
+    return opened;
+  }
+
+  public Date getClosed() {
+    return closed;
+  }
+
+  public Boolean getOpen() {
+    return isOpen;
+  }
+
+  public String getSentinel() {
+    return sentinel;
+  }
+
+  public Boolean getCancelInitiated() {
+    return cancelInitiated;
+  }
+
+  public Boolean getImmediateOrCancel() {
+    return immediateOrCancel;
+  }
+
+  public Boolean getConditional() {
+    return isConditional;
+  }
+
+  public String getCondition() {
+    return condition;
+  }
+
+  public String getConditionTarget() {
+    return conditionTarget;
+  }
+
+  @Override
+  public String toString() {
```

Comment:
toString seams no needed - may be lets remove them in those dtos?
such long ones don't help in debuging in any way and there not used in any other context

Reply:
I'm happy to remove it, if that's the design for dtos

Reply:
Removed